features/font-synthesis.yml.dist

```diff
@@ -2,9 +2,8 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: high
+  baseline: low
   baseline_low_date: 2022-01-06
-  baseline_high_date: 2024-07-06
```

Comment:
Even though this date is in the past, this change is rolled back, since the latest release in BCD is Safari 17.6, released 2024-06-17.